ci-operator/jobs/openstack-k8s-operators/ovn-operator/openstack-k8s-operators-ovn-operator-main-presubmits.yaml

```diff
@@ -236,7 +236,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
"always_run: true" was this marked to true by mistake?

Reply:
This need to be false, since we already have kuttl test running